### ISSUE TITLE
Dice Rolling is Wrong

### ISSUE THREAD

**Issue by jeffreycjohnson** [theloneplant/homestead-bot#10]
The bot gives equal probability to all outcomes between the min and max. That's not how statistics works. For 2d6 P(12) != P(7) for instance.

**Comment by theloneplant** [comment]
Good catch, the reason rand was used was for performance but a better way would be calculating each roll individually and setting a cap on the number of dice rolls allowed. It’s more work to add so idk when I’ll get around to it

**Comment by jeffreycjohnson** [comment]
For larger dice counts the results end up able to be approximated by the normal distribution, so after some cutoff you can switch to generating random numbers with a normal distribution instead of just erroring out.
